decorator.rb

```diff
@@ -0,0 +1,21 @@
+class Decorator
+  def initialize(nameable)
+    @nameable = nameable
+  end
+
+  def correct_name
+    @nameable
+  end
+end
+
+class CapitalizeDecorator < Decorator
+  def correct_name
+    @nameable.correct_name.capitalize
+  end
+end
+
+class TrimmerDecorator < Decorator
+  def correct_name
+    @nameable.correct_name.slice!(0, 5)
+  end
```

Comment:
- [ ] Kindly use the slice method to return 10 characters instead of 5 characters as the maximum limit for the name is 10 characters as per this [requirement](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/school_library_decorator_v2.md#:~:text=Implement%20a%20method%20correct_name%20that%20makes%20sure%20that%20the%20output%20of%20%40nameable.correct_name%20has%20a%20maximum%20of%2010%20characters).

Reply:
Thank you so much for noticing! I forgot to update that after doing some testing! Thanks again!
![OopsGIF](https://user-images.githubusercontent.com/50700379/210455677-88100104-df41-49bd-8923-edf8b847fcae.gif)